data/lang/ui-core/en.json

```diff
@@ -1095,6 +1095,10 @@
       "description" : "",
       "message" : "Ship is too small for the current crew."
    },
+   "TOO_SMALL_TO_TRANSSHIP" : {
+      "description" : "",
+      "message" : "Ship is too small to transship your cargo."
+   },
```

Comment:
"transship"? Shouldn't it be "transfer"?

If so, I can push change ASAP -- no need for new pull request.